packages/devtools_app/lib/src/framework/release_notes/release_notes.dart

```diff
@@ -20,6 +21,8 @@ import '../../shared/config_specific/server/server.dart' as server;
 import '../../shared/primitives/auto_dispose.dart';
 import '../../shared/theme.dart';
 
+final _log = Logger('lib/src/framework/release_notes/release_notes');
```

Comment:
using the full path will be painful when we end up refactoring things. What about using a logical name rather than a path name? e.g. "release notes", "framework core", "performance screen", etc